converter.js

```diff
@@ -113,9 +113,13 @@ const createConverter = config => {
         } else {
             rows.push(`export enum ${enum_.Identifier} {`);
 
-            entries.forEach(([key, value], i) => {
+            entries.forEach(([key, value]) => {
                 if (config.numericEnums) {
-                    rows.push(`    ${key} = ${value != null ? value : i},`);
+                    if (value == null) {
+                        rows.push(`    ${key},`);
+                    } else {
+                        rows.push(`    ${key} = ${value != null ? value : i},`);
```

Comment:
`value` is always `!= null` here, so this ternary expression can be removed

Reply:
Thats right! My mistake!
I did an ammend for previous commit.